### PR TITLE
Rename "Disabled Features" docs page

### PR DESCRIPTION
**Pull request by SuperQ**
Make the feature flags page more discoverable by naming it what it is.

Signed-off-by: SuperQ <superq@gmail.com>

### CODE REVIEW

File: docs/feature_flags.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
-title: Disabled Features
+title: Features Flags
```
Comment (RichiH) [inline docs/feature_flags.md:2]:
```suggestion
title: Feature Flags
```

**Review by RichiH — APPROVED** [review]
LGTM, one nit

File: docs/feature_flags.md (hunk 1 above)
Comment (SuperQ) [reply docs/feature_flags.md:2]: Whups

**Review by bwplotka — APPROVED** [review]
Thanks, much better 👍🏽

**Comment by LeviHarrison** [comment]
We also need to update the flag help text

https://github.com/prometheus/prometheus/blob/e98b639ac7d34d6ec6b8a4e4462b4bbe16cf7996/cmd/prometheus/main.go#L319

https://github.com/prometheus/prometheus/blob/e98b639ac7d34d6ec6b8a4e4462b4bbe16cf7996/cmd/promtool/main.go#L169

**Pull request merged**